examples/basic/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+deno run -c ./tsconfig.json -A --unstable ./mod.ts
```

Comment:
I, personally, think that we shouldn't be using `-A` or `--allow-all` instead of just adding `--allow-plugin --unstable` because we don't need all of those extra permissions and it adds insecurity as other modules would then be able to access all of that information.

Reply:
Also, there is another instance on line 38 of the README.

Reply:
You're right! I should have been more explicit.

Reply:
FYI: Allowing native plugins is just as insecure as `-A` — native plugins are not sandboxed, so it's the same as having all permissions for them. I still think being explicit is good though. Makes it easier for people to understand what's going on. Don't let yourself be cradled in a false sense of security: allowing plugins means all bets are off, and that's what the WASM version exists for.